core/common/src/main/java/alluxio/metrics/MetricsSystem.java

```diff
@@ -237,4 +254,23 @@ public void stop() {
   public MetricRegistry getMetricRegistry() {
     return mMetricRegistry;
   }
+
+  /**
+   * Util function to remove get the metrics name without instance and host.
+   * @param metricsName the long metrics name with instance and host name
+   * @return the metrics name without instance and host name
+   */
+  public static String stripInstanceAndHost(String metricsName) {
+    String[] pieces = metricsName.split("\\.");
```

Comment:
Instead of splitting and then joining later, would it be easier to follow if we tried pruning from the start of the string? (ie. check if master, prune `master.`, otherwise prune up to 2 `.`)